input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv

```diff
@@ -0,0 +1,107 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_ref, unit: TeV, datatype: float64}
+# - {name: e_ref_err, unit: TeV, datatype: float64}
+# - {name: dnde, unit: 1 / (cm2 s TeV), datatype: float64}
+# - {name: dnde_errn, unit: 1 / (cm2 s TeV), datatype: float64}
+# - {name: dnde_errp, unit: 1 / (cm2 s TeV), datatype: float64}
+# meta: !!omap
+# - {data_type: sed}
+# - {source_id: '49'}
+# - {reference_id: 2011ApJ...738...25A}
+e_ref e_ref_err dnde dnde_errn dnde_errp
+0.274999987355 0.0 1.70199886875e-09 2.49184145087e-11 2.49184145087e-11
+0.3399998734 0.0 1.28900222941e-09 1.27853577949e-11 1.27853577949e-11
+0.420001880993 0.0 8.82093557061e-10 1.07109914179e-11 1.07109914179e-11
+0.519001487363 0.0 5.77695603607e-10 9.95826611353e-12 9.95826611353e-12
+0.642000353315 0.0 3.50899394404e-10 9.69000585935e-12 9.69000585935e-12
+0.793001890196 0.0 2.15098893226e-10 9.58995409493e-12 9.58995409493e-12
+0.980000218484 0.0 1.30200144389e-10 9.83449608154e-12 9.83449608154e-12
+1.21199873196 0.0 6.27303291071e-11 8.98552643031e-12 8.98552643031e-12
+1.4980008244 0.0 3.3099930624e-11 8.6461757958e-12 8.6461757958e-12
+1.8510007513 0.0 1.66099892575e-11 8.22630908124e-12 8.22630908124e-12
+2.28800026098 0.0 1.12399967018e-11 9.06691573153e-12 9.06691573153e-12
+2.82799851617 0.0 6.15800716533e-12 9.1012436633e-12 9.1012436633e-12
+3.49599973938 0.0 3.34700164694e-12 9.07727690936e-12 9.07727690936e-12
+4.32098696108 0.0 1.16000723495e-12 7.52632406144e-12 7.52632406144e-12
+5.34200062268 0.0 5.22999528178e-13 6.76610843597e-12 6.76610843597e-12
+0.70099983322 0.0 3.87099976121e-10 2.06290005741e-11 2.06290005741e-11
```

Comment:
What do you mean? 
input/data/2011/2011ApJ...738...25A/tev-000049-sed.ecsv contains 22 lines, doesn't it?